apps/studio.giselles.ai/next.config.ts

```diff
@@ -105,6 +105,8 @@ const nextConfig: NextConfig = {
 	},
 	experimental: {
 		typedEnv: true,
+		webpackMemoryOptimizations: true,
+		webpackBuildWorker: true,
```

Comment:
I assume this is enabled since I don't have any custom webpack config.

> [Webpack build worker](https://nextjs.org/docs/app/guides/memory-usage#webpack-build-worker)
> The Webpack build worker allows you to run Webpack compilations inside a separate Node.js worker which will decrease memory usage of your application during builds.
>
> This option is enabled by default if your application does not have a custom Webpack configuration starting in v14.1.0.